Controller/ConfigController.php

```diff
@@ -220,7 +222,7 @@ public function deleteAction(Request $request, $id)
      */
     protected function createNewForm(Config $data, AdapterInterface $adapter)
     {
-        $form = $this->createForm('channel_config_new', $data, [
+        $form = $this->createForm($this->get('integrated_channel.form.config.new.type.proxy'), $data, [
```

Comment:
This is sollution should be incompatible for 3.0 if I am right so its better to just do it the right way now instead of later. Also if I am correct the new and edit are essential the same except for the name. This was dont this way so that it would be possibele to create seperate templetes for the edit en new form. As far as I know this is not used anywhere so just remove it and use the class name of the proxied config form.